test/puppetlabs/jdbc_util/pool_test.clj

```diff
@@ -120,7 +121,7 @@
                    pool/options->hikari-config)]
     (testing "if the init-fn throws an exception it continues to hand out connections normally"
       (let [wrapped (pool/connection-pool-with-delayed-init
-                     config (fn [_] (throw (RuntimeException. "test exception"))) 10000)]
+                     config {:migartion-db {}} (fn [_] (throw (RuntimeException. "test exception"))) 10000)]
```

Comment:
probably doesn't matter here but misspelled key in this and the next test change